src/Api/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -782,6 +781,7 @@ public async Task<OrganizationSsoResponseModel> PostSso(Guid id, [FromBody] Orga
 
     [HttpPut("{id}/collection-management")]
     [RequireFeature(FeatureFlagKeys.FlexibleCollections)]
+    [SelfHosted(NotSelfHostedOnly = true)]
```

Comment:
should the user be able to change this setting on the self-hosted instance?

Reply:
Only cloud access is required of this